sql-cli/sql_cli/__main__.py

```diff
@@ -38,5 +41,53 @@ def generate(
     rprint("The DAG file", dag_file.resolve(), "has been successfully generated. 🎉")
 
 
+@app.command()
+def init(project_dir: Optional[str] = typer.Argument(None)) -> None:
+    """
+    Initialise a SQL CLI project structure.
+
+    By default, this includes:
+
+    \b\n
+    ├── config
+    \b\n
+    │   ├── default
+    \b\n
+    │   │   └── configuration.yml
+    \b\n
+    │   └── dev
+    \b\n
+    │       └── configuration.yml
+    \b\n
+    ├── data
+    \b\n
+    │   ├── movies.db
+    \b\n
+    │   └── retail.db
+    \b\n
+    └── workflows
+    \b\n
+    ├── example_basic_transform
+    \b\n
+    │   └── top_animations.sql
+    \b\n
+    └── example_templating
+    \b\n
+        ├── filtered_orders.sql
+    \b\n
+        └── joint_orders_customers.sql
+
+    \b\n
+    Update the file `config/default/configuration.yaml` to declare your databases.
+    \b\n
+    Create SQL workflows within the `workflows` folder.
+    """
+    if project_dir is None:
+        project_dir = os.getcwd()
+
+    Project.initialise(Path(project_dir))
+    rprint(f"Initialized an Astro SQL project at {project_dir}")
```

Comment:
```suggestion
    rprint("Initialized an Astro SQL project at", project_dir)
```
I personally use `print(*args)`/`rprint(*args)` feature. You can pass all the values as separate args and print will join it for you and add a space inbetween.